nexus/src/db/datastore/resource_usage.rs

```diff
@@ -0,0 +1,221 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! [`DataStore`] methods on [`ResourceUsage`]s.
+
+use super::DataStore;
+use crate::context::OpContext;
+use crate::db;
+use crate::db::error::public_error_from_diesel_pool;
+use crate::db::error::ErrorHandler;
+use crate::db::model::ResourceUsage;
+use crate::db::pool::DbConnection;
+use crate::db::queries::resource_usage_update::ResourceUsageUpdate;
+use async_bb8_diesel::{AsyncRunQueryDsl, PoolError};
+use diesel::prelude::*;
+use omicron_common::api::external::DeleteResult;
+use omicron_common::api::external::Error;
+use oximeter::{types::Sample, Metric, MetricsError, Target};
+use std::sync::{Arc, Mutex};
+use uuid::Uuid;
+
+/// A collection which holds resources (such as a project, organization, or
+/// silo).
```

Comment:
```suggestion
/// Describes a collection that holds other resources.  Example targets might be a Project, Organization, or
/// Silo).
```

(I misread this at first, thinking "a project, organization, or silo" was one of the "resources")

Reply:
Fixed in 95bdaff87bab515331dce675a965d85f04126474